content/influxdb/cloud/notebooks/create-notebook.md

```diff
@@ -2,63 +2,144 @@
+5. Click the **+** icon, and then add or more of the following cell types to your notebook:
 
-  - [Downsample data](/influxdb/cloud/notebooks/downsample/)
-  - [Normalize data](/influxdb/cloud/notebooks/clean-data/)
+    - **Input cells**: to pull information into your notebook
+    - **Transform cells**: to filter and apply changes to your data
+    - **Pass-through**: to visualize and add notes to your data
+    - **Output**: to write data to a bucket, preview output before writing, or schedule an output task
 
-## Create a new notebook
-1. In the navigation menu on the left, click **Notebooks**.
+{{< tabs-wrapper >}}
+{{% tabs %}}
+[Input](#)
+[Transform](#)
+[Pass-through](#)
+[Output](#)
 
-    {{< nav-icon "notebooks" >}}
-2. Click **+Create Notebook**.
-3. Enter a name for your notebook in the **Name this notebook** field.
-4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](#cell-types) below for details.)
-5. Click the **+** icon to add a cell. See [Cell types](#cell-types) below for details on each type of cell.
-6. Click **Preview** to preview the results of each cell in a raw data table without writing any data.
-    {{% warn %}}
+<!-------------------------------- BEGIN Input -------------------------------->
+{{% tab-content %}}
+
+1. Select **Metric Builder** or **Query Builder** as your input, and then select the bucket to pull data from.
+2. Select filters to narrow your data.
+2. Select **Preview** or **Run** in the upper left dropdown menu.
+    - Click **Preview** (or press **CTRL + Enter**) to preview the results of each cell in a raw data table without writing any data. 
+    - Select **Run** to show the results of each cell and write it to the specified output bucket.
+
+{{% /tab-content %}}
+<!--------------------------------- END Input --------------------------------->
+
+<!-------------------------------- BEGIN Transform-------------------------------->
+{{% tab-content %}}
+
+1. Select one of the following transform cell-types:
+    - **Flux Script**: Use `__PREVIOUS_RESULT__` to build from data in the previous cell, enter a Flux script to transform your data. 
+    - **Downsample**: Window data by time and apply an aggregate to each window to downsample data. For more information, see [Downsample data with notebooks](/influxdb/cloud/notebooks/downsample/).
+2. Select **Preview** or **Run** in the upper left dropdown menu. By default, Preview appears.
+    - Click **Preview** (or press **CTRL + Enter**) to preview the results of each cell in a raw data table without writing any data. 
+    - Select **Run** to show the results of each cell and write it to the specified output bucket.
+
+   {{% warn %}}
 If your cell contains a custom script that uses any output function to write data to InfluxDB (the `to()` function) or send it to a 3rd party service, clicking **Preview** will write data.
```

Comment:
@mluu12 looks like this warning has a couple formatting issues with indent, space not around Preview with the bold, and to() is below the main line. Let's drop the backticks around the function and bold for now & indent text